### PR TITLE
Updated package.json Contributor list

### PR DESCRIPTION
**Pull request by mikr13**
This is to update the package.json file's contributor array to correctly mention Contributors since Github is not listing Contributors properly.

### CODE REVIEW

**Comment by butlerx** [comment]
could you please rebase of master

**Comment by mikr13** [comment]
I have resolved the conflicts of that file. One thing though, previously it was sorted in alphabetical order which was nice but you changed it, so there must be some purpose which I couldn't understand.

**Comment by butlerx** [comment]
The list was autogereated and ordered my number of commits I believe.

Could you please squash all your commits manually to ensure the right contributor is credited

**Review by butlerx — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -149,6 +150,7 @@
     "Matthew Piercey <piercey.matthew@gmail.com>",
     "Neale Pickett <neale@woozle.org>",
     "Robert <robert@n5qm.com>",
+    "Soura Dutta <duttasoura@gmail.com>"
```
Comment (butlerx) [inline package.json]: You appear to be missing a comma

**Review by butlerx — CHANGES_REQUESTED** [review]

**Review by butlerx — CHANGES_REQUESTED** [review]
If you could fix the comma I can merge this

File: package.json (hunk 1 above)
Comment (MIKR13MS) [reply package.json]: Sure, I'll fix that.

Comment (MIKR13MS) [reply package.json]: It's done, the comma is there now. Sorry got a little busy with some other projects.

**Pull request merged**